lib/python/ray/worker.py

```diff
@@ -674,21 +691,26 @@ def get_address_info_from_redis(redis_address, node_ip_address, num_retries=5):
       time.sleep(1)
     counter += 1
 
-def init(node_ip_address=None, redis_address=None, start_ray_local=False, object_id_seed=None, num_workers=None, num_local_schedulers=None, driver_mode=SCRIPT_MODE):
-  """Either connect to an existing Ray cluster or start one and connect to it.
+def _init(address_info=None, start_ray_local=False, object_id_seed=None,
+          num_workers=None, num_local_schedulers=None,
+          driver_mode=SCRIPT_MODE):
+  """Helper method to either connect to an existing Ray cluster or start one
```

Comment:
The first doc string line is supposed to fit in one line. Maybe  `"""Helper method to connect to an existing Ray cluster or start a new one.`